app/templates/home.html

```diff
@@ -1,24 +1,11 @@
-{% extends "partials/index/index_base.html" %}
+{% extends "base" %}
```

Comment:
Sure?

Reply:
This file is no longer in use. I'm removing it from this pull request as well.